DxDispatch/models/_schema.json

```diff
@@ -76,6 +76,15 @@
             ]
         },
 
+        "DmlCompileType":
```

Comment:
nit: `dmlCompileType`. Everything else is camelCase, not PascalCase.

Reply:
BufferViewType, PixCaptureType, Compiler, TokenType are all PascalCase, I can update to camelCase if that is preferred though

Reply:
Everything in the schema is camelCase. That doesn't mean everything in the entire code base is this way.